docs/reference/schemas/config/functions/int.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Reference for the 'int' DSC configuration document function
+ms.date:     04/09/2024
+ms.topic:    reference
+title:       int
+---
+
+# int
+
+## Synopsis
+
+Returns an integer from an input string or non-integer number.
```

Comment:
I'm not sure if `int()` needs to be capable of converting from a _non-integer number_, as https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-numeric#int specifies the input as `string` or `int` (...though I can't really think of a use case for calling `int()` with an `int`)

Reply:
I think there's some value in getting return data from a resource like `1.3` and converting it to `1`, but I don't fully disagree with you.

It's also worth noting that, right now, converting a number with a fractional part to an integer with this configuration function is broken - see #390 for more details.

Reply:
@michaeltlombardi can you open an issue to track this?